terraform/modules/r53-resolver-logs/main.tf

```diff
@@ -1,10 +1,38 @@
+data "aws_caller_identity" "current" {}
+
 resource "aws_route53_resolver_query_log_config" "main" {
-  name            = "dns_logs"
-  destination_arn = var.logging_destination_arn
+  name            = format("%s-r53-resolver-logs", var.vpc_id)
```

Comment:
Perhaps `var.vpc_name` could be a more user friendly name? Although I think that would require more changes

Reply:
It would be friendlier, yeah. I think I can accommodate that. It should be a matter of supplying it through the `for_each` that I have in place.